### ISSUE TITLE
Runtime org.kde.Platform branch 6.4 is end-of-life

### ISSUE THREAD

**Issue by jakobjakobson13** [flathub/org.tug.texworks#35]
The kde.Platform branch 6.4 is end-of-life and should be updated.

```
ich@localhost:~> flatpak update
Suche nach Aktualisierungen …

Info: runtime org.kde.Platform branch 6.4 is end-of-life, with reason:
   We strongly recommend moving to the latest stable version of the Plaform and SDK
Info: applications using this runtime:
   org.tug.texworks

```

**Comment by stloeffler** [comment]
Unfortunately, texworks 0.6.8 doesn't work with Qt 6.5 (or later). This will be fixed in the upcoming texworks 0.6.9, until then I would stay at Qt 6.4 (rather than starting cherry-picking a bunch of patches).

**Comment by stloeffler** [comment]
Fixed in be5fc113692a77d5b8e76b20eedfca3a2e42c173
